plone/formwidget/namedfile/tests.py

```diff
@@ -1,16 +1,55 @@
 # -*- coding: utf-8 -*-
-from z3c.form import testing
 import doctest
 import unittest
+# import interlude
+
+
+def setUp(self):
```

Comment:
The custom setup here is necessary, since the one from z3c.form.testing overwrites the globals.